app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
Seems ok to me. Seems like there's a precedent for using it. Would like to know what @jrafanie and @Fryguy think...